pyspider/database/elasticsearch/resultdb.py

```diff
@@ -36,7 +36,7 @@ def projects(self):
         ret = self.es.search(index=self.index, doc_type=self.__type__,
                              body={"aggs": {"projects": {
                                  "terms": {"field": "project"}
-                             }}}, _source=False)
```

Comment:
Seems the API changed to `source`, that this API is meant to return just project names, that the source doesn't matter.
https://elasticsearch-py.readthedocs.io/en/latest/api.html#elasticsearch.Elasticsearch.search